src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -260,6 +261,132 @@ public static void BuildChain_WithCertificatePolicy_NoMatch()
         }
 
         [Fact]
+        public static void BuildChain_MicrosoftDotCom_WithSystemRootCerts()
+        {
+            // Verifies that the microsoft.com certs build with only the certificates in the root store
+            // This is a canary test for BuildChain_MicrosoftDotCom_WithRootCertInUserAndSystemRootCertStores
+
+            using (var microsoftDotCom = new X509Certificate2(TestData.MicrosoftDotComSslCertBytes))
+            using (var chainHolder = new ChainHolder())
+            {
+                X509Chain chainValidator = chainHolder.Chain;
+                bool chainBuildResult = chainValidator.Build(microsoftDotCom);
+
+                StringBuilder builder = new StringBuilder();
+                foreach (var status in chainValidator.ChainStatus)
+                {
+                    builder.AppendFormat("{0} {1}{2}", status.Status, status.StatusInformation, Environment.NewLine);
+                }
+
+                Assert.True(chainBuildResult, "Chain build for microsoft.com cert failed. Reason: " + builder.ToString());
+            }
+        }
```

Comment:
@bartonjs - do you think this canary is needed? I don't see another test that does the same type of testing, so I thought I'd add this as a sort of canary instead of trying to test it in BuildChain_MicrosoftDotCom_WithRootCertInUserAndSystemRootCertStores